Monet_Harkin/Survey3/apps/survey_form/views.py

```diff
@@ -0,0 +1,25 @@
+from django.shortcuts import render, redirect
+
+
+# Create your views here.
+def index(request):
+	if "count" not in request.session:
+		request.session["count"]=0
+	request.session["name"]=""
+	request.session["location"] = ""
+	request.session["language"] = ""
+	request.session["comment"] = ""
+	return render(request, "survey_form/index.html")
```

Comment:
Since you're not using any session data on your index.html you don't need to create all of the request.session... keys here